flexibleengine/resource_flexibleengine_cce_node_v3.go

```diff
@@ -21,6 +22,18 @@ func resourceCCENodeV3() *schema.Resource {
 		Read:   resourceCCENodeV3Read,
 		Update: resourceCCENodeV3Update,
 		Delete: resourceCCENodeV3Delete,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				// https://www.terraform.io/docs/extend/resources/import.html#importer-state-function
+				arr := strings.Split(d.Id(), ":")
```

Comment:
according to [other resource](https://www.terraform.io/docs/providers/flexibleengine/r/lb_l7rule_v2.html), we'd better use the slash (/) to split cluster id and node id.

the website also should be updated as well.

Reply:
website removed. Thanx